src/tables/table.py

```diff
@@ -257,7 +259,7 @@ def _query_table(self, sql):
         df = None
         self._print(sql)
         try:
-            df = pandas.read_sql(sql, self._engine, index_col=self._index_col)
+            df = pandas.read_sql(sql, self._engine)
 
```

Comment:
Hasn't this change caused us grief in the past?

Reply:
I don't think I made it?